src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -268,8 +268,8 @@ void _port_PropertyChanged(object sender, System.ComponentModel.PropertyChangedE
                 case "Center":
                     RaisePropertyChanged("Center");
                     break;
-                case "DefaultValueEnabled":
```

Comment:
`DefaultValueEnabled` was removed on the `NodeModel`, so the change event can be removed here. The property was removed from `NodeModel` in favor of using the more precise `DefaultValue == null`.